gcn/notices/svom/mxt/localization.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/mxt/localization.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM/MXT localisation parameters",
+  "description": "MXT refined localisation after a slew of the SVOM platform on an ECLAIRs triggered source",
+  "allOf": [
+    { "$ref": "../../core/AdditionalInfo.schema.json" },
+    { "$ref": "../../core/Alert.schema.json" },
+    { "$ref": "../../core/DateTime.schema.json" },
+    { "$ref": "../../core/Event.schema.json" },
+    { "$ref": "../../core/Localization.schema.json" },
+    { "$ref": "../../core/Reporter.schema.json" },
+    { "$ref": "../../core/Statistics.schema.json" }
+  ],
+  "properties": {
+    "$schema": true,
+    "mean_flux": {
+      "type": "number",
+      "description": "Mean flux of the detection [erg.cm^-2]"
```

Comment:
Flux units are (erg cm^-2 s^-1), we have energy_flux property defined in our core schema (https://raw.githubusercontent.com/nasa-gcn/gcn-schema/main/gcn/notices/core/spectral/GammaRay.schema.json), might be useful here.
erg cm^-2 is units for fluence.